packages/SystemsThinkingWorkbench-Core.package/STWNumberMorph.class/instance/interimContents..st

```diff
@@ -1,6 +1,6 @@
 editing
 interimContents: aString
 
-	(aString isAllDigits and: aString ~= '')
+	(aString isAllDigits and: aString ~= '' and: (self maximum ifNil: [true] ifNotNil: [aString asNumber <= self maximum]))
```

Comment:
Maybe add a help method called `node>>canHold: aNumber` to make this actually readable.